video/cloud-client/analyze/beta_snippets.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This application demonstrates face detection, label detection,
+explicit content, and shot change detection using the Google Cloud API.
+
+Usage Examples:
+    python beta_snippets.py boxes \
+    gs://python-docs-samples-tests/video/googlework_short.mp4
+
+    python beta_snippets.py \
+    emotions gs://python-docs-samples-tests/video/googlework_short.mp4
+
+    python beta_snippets.py \
+    transcription gs://python-docs-samples-tests/video/googlework_short.mp4
+"""
+
+import argparse
+
+from google.cloud import videointelligence_v1p1beta1 as videointelligence
+
+
+# [START video_face_bounding_boxes]
+def face_bounding_boxes(path):
```

Comment:
Can they use a local file too or just a GCS file?

Reply:
They can.  The request needs to be only slightly different, such as https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/video/cloud-client/analyze/analyze.py#L134

Reply:
Do we only need to provide local file snippets, no GCS snippets?

Reply:
Here we are showing only GCS snippets.

Reply:
OH, do Python samples all use `path` for GCS? I figured that was for a local file

It should be `uri`, based on existing samples, eg: https://cloud.google.com/vision/docs/detecting-labels#vision-label-detection-python

```python
def detect_labels(path):
    """Detects labels in the file."""
    client = vision.ImageAnnotatorClient()
```

```python
def detect_labels_uri(uri):
    """Detects labels in the file located in Google Cloud Storage or on the
    Web."""
    client = vision.ImageAnnotatorClient()
```

Reply:
Let's make this change across the board to all 3 samples

Reply:
you are right - it should be `uri`, or rather `gcs_uri` (since currently the API only supports that).  fixed.